src/main/java/org/junitpioneer/jupiter/RangeSource.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.params.provider.ArgumentsSource;
+
+/**
+ * {@code @RangeSource} is an {@link ArgumentsSource} which provides access to a range of {@code int} values.
+ *
+ * The supplied values will be provided as arguments to the annotated {@code @ParameterizedTest} method.
+ *
+ * @see ArgumentsSource
```

Comment:
Once we know which version this lands in, we need a `@since` tag.

Reply:
I'll add a `@since 4.0` annotation as a placeholder in the meanwhile

Reply:
I unresolved this, so we keep track of it.